core-bundle/src/Framework/ContaoFramework.php

```diff
@@ -114,6 +117,8 @@ public function initialize(bool $isFrontend = false): void
         if (!$this->legacyRouting) {
             $this->throwOnLegacyRoutingHooks();
         }
+
+        self::$nonce = bin2hex(random_bytes(16));
```

Comment:
We could move this into the `getNonce()` method so we only generate the nonce if someone uses it.

```php
public static function getNonce(): string
{
    if ('' === self::$nonce) {
        self::$nonce = bin2hex(random_bytes(16));
    }

    return self::$nonce;
}
```

Reply:
Done in 015450554.